### DISCUSSION TITLE
ID or Permalink visible in GUI

### DISCUSSION THREAD

**Discussion by TheOrganist24** [jonaswinkler/paperless-ng#798] [Feature Requests]
This is a great piece of tech which I am hoping to use as a source of truth for all my documents. What would be great is to be able to easily grab a permalink, or at least an ID from the GUI. That way I can store references as part of other projects.

I realise a lot of this can be done easily through the Rest API, however I think it would be useful to have a link displayed in perhaps in the listed tiles view underneath the document next to the download button. I realise it would be only a cosmetic change since you can achieve the same through right-clicking the download button and grabbing the link that way. I guess I can just see how sometimes it's useful to know at least what the ID of the document is.

**Comment by shamoon** [comment]
Hmm, real estate is quite tight on the cards and we just added ASNs & doc types (#748). The ID really doesnt have any real world meaning other than how Paperless stores its documents so Im just not sure displaying has much value for the vast majority of folks... Also, sounds like with right-clicking to get a permalink you have a pretty good solution for your use case. Just my opinion obv, not sure if others feel differently.
